sheetstorm/management/commands/generate_test_data.py

```diff
@@ -26,7 +30,7 @@ class Command(BaseCommand):
     help = "Create initial sample data for SheetStorm application testing."
 
     @transaction.atomic
-    def handle(self, *args, **options):
+    def handle(self, *args: Any, **options: Union[bool, int, None]) -> None:
```

Comment:
You're using everywhere else for `options` typing like this: `Dict[str, Union[bool, int, None]]`. Why haven't  you used here it also?

Reply:
There is only need to type annotation of potential values in **kwargs, and options will be `Dict[str, Union[bool, int, None]]`. If I will  write here a Dict type, I will get `Dict[str, Dict[str, Union[bool, int, None]]]` instead.